src/node.cc

```diff
@@ -432,6 +432,9 @@ void ResetSignalHandlers() {
 }
 
 static std::atomic<uint32_t> init_process_flags = 0;
```

Comment:
Maybe a comment here describing why we use a `uint32_t`?

Reply:
Yeah, makes sense – done in c259b3b3aecfa43af4db190b78094690f2faeef4!